contracts/protocol/clients/voucher/BosonVoucher.sol

```diff
@@ -688,80 +743,76 @@ contract BosonVoucherBase is IBosonVoucher, BeaconClientBase, OwnableUpgradeable
         uint256 _tokenId
     ) internal override {
         if (_premintStatus.committable) {
-            // If is committable, invoke commitToPreMintedOffer on the protocol
-
-            // Store offerId so _premintStatus can be deleted before making an external call
-            uint256 offerId = _premintStatus.offerId;
+            // Store range owner so _premintStatus can be deleted before making an external call
+            address rangeOwner = _premintStatus.owner;
```

Comment:
This is reading from the storage.
On the other hand  `_rangeByOfferId[_tokenId >> 128].owner` is slightly more expensive, but it does not require writing to `_premintStatus.owner` in the first place.

It might be sensible to verify if using `_premintStatus.owner` is reasonable at all.
Moreover, in PRs that are coming to 2.2., `_premintStatus` is already removed and only `isCommitable` is used instead.

Reply:
makes sense, removed